### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/technical-classification/data-dates/data_collection_start

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#54]
**Describe the request**  
Adding wording if the collection/creation of data has already started

**Original setting or content**   

*question text ger*  
"Wann beginnt die Erhebung bzw. Erstellung der Daten?"

**Proposed setting or content**   

*question text ger*  
"Wann beginnt/begann die Erhebung bzw. Erstellung der Daten?"
*question text en*
"When does data collection or creation start/begin?"

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] question text (en and ger)

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/76

**Comment by ClaraRosalie** [comment]
implemented in upstream rdmorganiser#86
